config/examples/Kingroon/KP3/Configuration.h

```diff
@@ -89,7 +89,7 @@
 
 // Choose the name from boards.h that matches your setup
 #ifndef MOTHERBOARD
-  #define MOTHERBOARD BOARD_MKS_ROBIN_MINI
+  #define MOTHERBOARD BOARD_MKS_TINY_BEE
```

Comment:
Are you sure this is correct? From what I can find, `BOARD_MKS_ROBIN_MINI` is the correct motherboard for this printer.